### PR TITLE
Port and update snippets content from DCC

### PR DESCRIPTION
**Pull request by kaycebasques**
@Meggin or @pbakaus please review

* Ported old [snippets content](https://developer.chrome.com/devtools/docs/authoring-development-workflow#snippets) from DCC.
* Added some nifty animations.

### CODE REVIEW

**Comment by petele** [comment]
@pbakaus - where should this go in the new IA?

**Comment by pbakaus** [comment]
This doesn't really belong into the getting started guide. It sits best in "Debug" as high level item or folded into "Interact from Command Line".

@kaycebasques can you move it to the right section and prepare the pull request against the material-branch?

**Comment by kaycebasques** [comment]
Hallo @pbakaus, can do. I'll be out today thru Fri. Can work on this on Mon. Does the content look :ok_hand: ?

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+authors:
+  - kaycebasques
+article:
+  written_on: 2015-10-01
+  updated_on: 2015-10-01
+  order: 1
+collection: workspace
+takeaways:
+    - Use snippets to run small scripts from any page.
+    - Run portions of snippets in the console with the "Evaluate in
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: console -> Console

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      Console" feature.
+    - Map snippets to external files with workspaces.
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: workspaces -> Workspaces (it's a name)

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+and delete snippets.
+
+1. [Open DevTools](setup-devtools#access-devtools) and go to the 
+   **Sources** panel.
+
+1. Click the layout toggle in the top-left corner of DevTools.
+
+1. Click on the **Snippets** panel.
+
+The video below shows how to navigate to the snippets panel. The **Sources**
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: This whole paragraph is overkill. No need to describe what's happening in the video. Also, probably don't need a video for this. Videos I usually reserve for something like showing a workflow in action.

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+## Create snippets
+
+The left side of the snippets panel displays the list of snippets available
+on your machine. The right panel displays an editor which you can use to
+create and modify snippets. Once you have created snippets, clicking on
+a snippet filename in the left panel makes DevTools display the 
+snippet's source code in the right panel.
+
+1. Right-click inside the file list panel.
+
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: No empty lines between lists

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+1. Select **New** to create a new snippet file.
+
+1. Type in a filename for your snippet, or type `Enter` to use the
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: type "Enter" reads like I should type the word Enter somewhere :) Maybe "press Enter"? Just a nit though, feel free to ignore.

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+
+1. To save, right-click in the source code panel and select **Save**. Or,
+   type `Command` + `S` on Mac, or `Control` + `S` on Windows. You should
+   see the asterisk in front of the snippet filename disappear.
+
+In the video below, a snippet is created by right-clicking in the
+file list panel and selecting **New**. Chrome DevTools automatically names
+the file `Script snippet #11`. The filename is changed to `hi.js`. A 
+line of code is entered in the editor. The file is saved by right-clicking
+in the editor and selecting **Save**.
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: Same as above, don't describe the content of the video.

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+
+![run snippet by right-clicking on filename](imgs/run-snippet-1.png)
+
+* Click the **Run** button, highlighted in red in the image below.
+
+![run snippet by clicking "run" button](imgs/run-snippet-2.png)
+
+* Type `Command` + `Enter` on Mac, or `Control` + `Enter` on Windows. 
+
+### Evaluate in console
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: console -> Console

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+
+You can also evaluate a portion of a snippet in the DevTools console. 
+
+1. Highlight the portion of code which you wish to run.
+
+1. Right-click anywhere in the editor and select **Evaluate in Console**.
+   Or, use the keyboard shortcut `Control` + `Shift` + `E` for Windows, or
+   `Command` + `Shift` + `E` for Mac.
+
+In the video below, a portion of a snippet is highlighted, and then that
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: Same as above, skip the description.

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+portion is evaluated by right-clicking on the editor and selecting 
+**Evaluate in Console**. You can see in the console that only the statements 
+that were highlighted are evaluated. The statements that were not highlighted 
+are not evaluted.
+
+{% animation animations/evaluate-in-console.mp4 %}
+
+## Map snippets to external files via workspaces
+
+By default, snippets are saved to DevTools local storage, which is not
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: DevTools should probably be DevTools' , no?

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+easy to access via external editors. If you wish to edit your snippets
+with external editors, or to use revision control systems like Git
+to track the history of your snippets, you can map your snippets to
+external files using [workspaces](setup-workflow).
```
Comment (pbakaus) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: workspaces -> Workspaces

**Comment by pbakaus** [comment]
@kaycebasques sounds good. Left a few comments, generally looks great!

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 1 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: :heavy_check_mark:

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 2 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: :heavy_check_mark:

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 4 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: :heavy_check_mark:

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 5 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: :heavy_check_mark:

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 6 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: Actually @Meggin asked me to include descriptions of videos. I personally think it's good practice from an accessibility perspective.

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 7 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: :heavy_check_mark:

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 8 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: See note above.

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 3 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: See note below.

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 9 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: :heavy_check_mark:

File: src/_langs/en/tools/setup/workspace/setup-snippets.markdown (hunk 10 above)
Comment (kaycebasques) [inline src/_langs/en/tools/setup/workspace/setup-snippets.markdown]: :heavy_check_mark:

**Comment by kaycebasques** [comment]
Continued in PR #2113

**Pull request closed without merging**